### PR TITLE
Issue #2902897 by Makishima: Improve usability of event overview

### PR DESCRIPTION
**Pull request by VolodymyrMartsynenko**
**Related issue:** https://www.drupal.org/node/2902897

**Changes:**
- Replace 'Events that have started or finished' for 'Past events'.
- current events should be shown under the upcoming event filter
- end date should be mandatory
- after end date past relocate item to past events

### CODE REVIEW

**Comment by ribel** [comment]
@VolodymyrMartsynenko, please also replace string  **Events that have started or are finished** with **Past events** in behat tests, and check logic of **homepage-community-events.feature**

**Comment by MaikelGG** [comment]
@VolodymyrMartsynenko Can you solve the merge conflict in social_event.info.yml?

**Comment by MaikelGG** [comment]
Notice: Undefined index: field_event_date_value_op in Drupal\social_event\Plugin\views\sort\EventPassedDesc->query() (line 18 of profiles/contrib/social/modules/social_features/social_event/src/Plugin/views/sort/EventPassedDesc.php).

**Comment by jochemvn** [comment]
@VolodymyrMartsynenko, could you check the remark that @MaikelGG made? Also could you check why the builds are failing?

File: tests/behat/features/capabilities/event/homepage-community-events.feature [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@api @event @stability @perfect @community @upcoming @overview @block @LU @critical @DS-1056 @stability-2
+@api @event @stability @perfect @community @upcoming @overview @block @LU @critical @DS-1056 @stability-2 @mytest
```
Comment (bramtenhove) [inline tests/behat/features/capabilities/event/homepage-community-events.feature]: This tag can be removed.

**Comment by jochemvn** [comment]
Sorting of the event overview looks changed as well. Please take a look at it: https://screencast.com/t/KbrsEhtlBA

**Comment by jochemvn** [comment]
@VolodymyrMartsynenko do you have a status update on the above?

**Comment by bramtenhove** [comment]
@VolodymyrMartsynenko do you have any update on this Pull Request?

**Comment by jaapjan** [comment]
@ribel Do you perhaps know what we should do with this Pullrequest? Looks like it still needs work?

**Pull request closed without merging**